monai/transforms/intensity/array.py

```diff
@@ -461,8 +471,10 @@ class ScaleIntensityRangePercentiles(Transform):
     def __init__(
         self, lower: float, upper: float, b_min: float, b_max: float, clip: bool = False, relative: bool = False
     ) -> None:
-        assert 0.0 <= lower <= 100.0, "Percentiles must be in the range [0, 100]"
-        assert 0.0 <= upper <= 100.0, "Percentiles must be in the range [0, 100]"
+        if 0.0 > lower:
+            raise AssertionError("Percentiles must be in the range [0, 100]")
```

Comment:
the autofix isn't quite right, how to report this kind of issues? @srijan-deepsource

Reply:
@wyli Thank you for bringing it to my attention. We'll look into this behavior.

> how to report this kind of issues?

We have a [Slack channel](http://deepsourceusers.slack.com/) where you can report bugs you come across.